.github/workflows/ci-cd-payment-service.yml

```
@@ -32,6 +33,11 @@ jobs:
         java-version: '21'
         distribution: 'temurin'
 
+    - name: Set up application.properties
+      run : |
```

YAMLlint (1.37.1)

[warning] too many spaces before colon

(colons)